GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using UnityEngine;
+using UnityEngine.Purchasing;
+
+public class PurchaseController : MonoBehaviour, IStoreListener
+{
+    private static IStoreController m_StoreController; // The Unity Purchasing system.
+    private static IExtensionProvider m_StoreExtensionProvider; // The store-specific Purchasing subsystems.
+
+    private void Start()
+    {
+        // If we haven't set up the Unity Purchasing reference
+        if (m_StoreController == null)
+        {
+            // Begin to configure our connection to Purchasing
+            InitializePurchasing();
+        }
+    }
+
+    private void InitializePurchasing()
+    {
+        // If we have already connected to Purchasing ...
+        if (IsInitialized())
+        {
+            // ... we are done here.
+            return;
+        }
+
+        // Create a builder, first passing in a suite of google play store.
+        var builder = ConfigurationBuilder.Instance(Google.Play.Billing.GooglePlayStoreModule.Instance());
+
+        // Add consumable products (coins) to sell / restore by way of its identifier,
+        // associating the general identifier with its store-specific identifiers.
+        foreach (var coin in CoinList.List)
+        {
+            builder.AddProduct(coin.ProductId, ProductType.Consumable);
+        }
+
+        // Continue adding the non-consumable products (car).
+        foreach (var car in CarList.List)
+        {
+            if (car.IsPriceInDollar && car.Price > 0)
```

Comment:
IsRealMoneyPurchase might be more clear than IsPriceInDollar

Reply:
good idea! I will take that.